design/notifications.md

```diff
@@ -0,0 +1,360 @@
+
+# Notifications
+This document outlines the design for notifications in Nessie with an application for Webhooks. It outlines the different
+events that we support, the way the notifications are called, how they are stored and what is the API used for creating,
+listing, updating and deleting notifications for a nessie server.
+
+## Event types
+This section lists out all events we support. All events in Nessie contain two pieces of information: the event time and
```

Comment:
nit: maybe make this a list so that it's immediately visible that there is event time +  event type